.github/workflows/vagrant-packet.yaml

```diff
@@ -14,4 +14,4 @@ jobs:
     - name: Vagrant Test
       run: |
         export VAGRANT_DEFAULT_PROVIDER="virtualbox"
-        go test --timeout 1h -v ./test/_vagrant
+        go test --timeout 1h -v -count 1 ./test/_vagrant
```

Comment:
This belongs in its own commit. Actually why is this even needed? The tests should not be cached in GHA right?

Reply:
I have added this because apparently they are. I think the self-managed runner we have is caching them

Reply:
I will dig into it separately.

Reply:
ah!